packages/client/src/helpers/client/interceptors/authentication/AxiosAuthenticationTokenManager.ts

```diff
@@ -18,15 +19,39 @@ import {
   UserSessionExpiredError,
 } from './errors';
 import AuthenticationConfigOptions from './AuthenticationConfigOptions';
-import axios from 'axios';
+import axios, { AxiosError, AxiosInstance, AxiosResponse } from 'axios';
 import isNil from 'lodash/isNil';
+import type { Data } from './token-providers/types/TokenData.types';
+import type {
+  OptionsStorageProvider,
+  OptionsStorageSerializer,
+  Options as TokenManagerOptions,
+} from './types/TokenManagerOptions.types';
+import type { RequestConfig } from './types/AxiosAuthenticationTokenManager.types';
+import type UserToken from './types/UserToken.types';
 
 /**
  * Class responsible for installing an axios interceptor which will manage
  * authentication of any requests that require it with the correct
  * access token type (user or guest).
  */
 class AxiosAuthenticationTokenManager {
+  activeTokenDataChangedListener:
+    | ((activeToken: { kind: TokenKinds; data: Data | null } | null) => void)
+    | null;
+  authorizationHeaderFormatter!: TokenManagerOptions['authorizationHeaderFormatter'];
+  axiosInstance!: AxiosInstance;
+  clientCredentialsTokenProvider!: ClientCredentialsTokenProvider;
+  currentTokenProvider: UserTokenProvider | GuestTokenProvider | null;
+  guestTokenProvider!: GuestTokenProvider;
+  isLoaded: boolean;
+  isLoading: boolean;
+  loadError: MisconfiguredTokenProviderError | null;
+  loadPromise: Promise<unknown> | null;
```

Comment:
In this case we used unknown, because with void we get an error when using loadPromise.
`Type 'Promise<(null | undefined)[]>' is not assignable to type 'Promise<void>' `

Reply:
My mistake, resolving it. Thanks!